00-RELEASENOTES

```diff
@@ -13,498 +13,20 @@ SECURITY: There are security fixes in the release.
-  enable loading the file in the absence of the module (#11374)
-* Add a dry run flag to RM_Call to do validations before actual execution (#11158)
-* Add RM_Microseconds and RM_CachedMicroseconds (#11016)
-* Add RM_ACLAddLogEntryByUserName API to be used without a user object (#11659)
-* Make it possible to keep the RM_Call reply for longer than the context lifetime in case
-  auto memory was not used (#11568)
-
-Potentially Breaking Changes in Module API
-------------------------------------------
-
-* RM_Call only enforces OOM on scripts if 'M' flag is set (#11425)
-* Block some specific characters in module command names (#11434)
-* Fix replication inconsistency on modules that uses keyspace notifications (#10969)
-* Prevent command, configs, data types registration after the onload handler (#11708)
-
-Bug Fixes
-=========
-
-* Introduce socket shutdown to properly disconnect a client while a fork is active (#11376)
-* CLIENT RESET clears the CLIENT NO-EVICT flag (#11483)
-* Reduce memory usage on strings loaded by a module from an RDB file (#11050)
-* Fix a bug where nodes in a cluster may not replicate or handle internal events for
-  keys deleted when another node in the cluster claimed a slot (#11084)
-* Fix HINCRBYFLOAT not to create a key if the new value is invalid (#11149)
-* Make cluster config file saving atomic and fsync acl file saving (#10924)
-* WAIT command would not block if used in RM_Call (#11713)
-* Minor fixes to command metadata in COMMAND command (#11201, #10273)
-
-
-Thanks to all the users and developers who made this release possible.
-We'll follow up with more RC releases, until the code looks production ready
-and we don't get reports of serious issues for a while.
-
-A special thank you for the amount of work put into this release by:
-
-- Meir Shpilraien
-- Guy Benoish
-- Viktor Söderqvist
-- Zhu Binbin
-- Oran Agra
-- sundb
-- Ran Shidlansik
-- Zhenwei Pi
-- Jason Elbaum
-- Karthik Subbarao
-- Madelyn Olson
-- Huang Zhw
-- Ping Xie
-- Ozan Tezcan
-- Chen Tianjie
-- Deng Ju
-- Wen Hui
-- Brennan Cathcart
-- Itamar Haber
-- Shaya Potter
-- Roshan Khatri
-- Slava Koyfman
-- Zhu Tian
-- Moti Cohen
-- Arad Zilberstein
-- Basel Naamna
-- Mingyi Kang
-- Uri Yagelnik
-- Filipe Oliveira
-- Zhao Zhao
-- Valentino Geron
-- Yaacov Hazan
-- Adi Pinsky
-- David Carlier
-- Li Changjun
-
+* Rename the 6 Redis binaries to ones with valkey prefixes: `valkey-server`, `valkey-cli`, `valkey-benchmark`, `valkey-check-aof`, `valkey-check-rdb` and `valkey-sentinel`. (#62)
+* During install, add support for making symlinks mapping from the corresponding Redis binaries to the new Valkey binaries. (#193)
+* Introduce `valkey-version` and `sever-name` as info fields. The `redis-version` will indicate the OSS Redis version this valkey version is compatible with. (#47)
+* Introduce `valkey-ver` RDB field to indicate an RDB field was produced by a Valkey instance. (#47)
```

Comment:
i see in #47, it is `server-ver`?

Reply:
This should also include #232 I suppose, which includes the rename from server-version -> valkey-version